packages/Panel/README.md

```diff
@@ -22,23 +22,24 @@ npm install @paprika/panel
+| isCompact             | bool                                                                                                 | false    | false                           | Control the compactness of the Panel                                                                                                            |
+| isInline              | bool                                                                                                 | false    | false                           | Render the panel inline                                                                                                                         |
+| isOpen                | bool                                                                                                 | true     | -                               | Control the visibility of the Panel. This prop makes the Panel appear.                                                                          |
+| kind                  | [ Panel.types.kind.DEFAULT, Panel.types.kind.CHILD, Panel.types.kind.PRIMARY]                        | false    | Panel.types.kind.DEFAULT        | Modify the look of the Panel                                                                                                                    |
+| offset                | shape                                                                                                | false    | { top: 0, left: 0, right: 0 }   | Control offset of the Panel. Only use 'top' when sliding in from the left or right. Only use 'left' or 'right' when sliding in from the bottom. |
+| onAfterClose          | func                                                                                                 | false    | () => {}                        | Callback once the Panel has been closed event                                                                                                   |
+| onAfterOpen           | func                                                                                                 | false    | () => {}                        | Callback once the Panel has been opened event                                                                                                   |
+| onClose               | func                                                                                                 | false    | null                            | Callback triggered when the Panel needs to be close                                                                                             |
+| slideFrom             | [ Panel.slideFromDirections.RIGHT, Panel.slideFromDirections.LEFT, Panel.slideFromDirections.BOTTOM] | false    | Panel.slideFromDirections.RIGHT | Control where the Panel slides in from                                                                                                          |
```

Comment:
I think you need to re-run generatReadme, also it should be:

`types.slideFroms`=> `types.slideFrom` because you match the prop name